integration/tests/services/helpers/basic.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+import json
+import typing
+from types import SimpleNamespace
+
+from eth_utils import keccak
+from websockets.legacy.client import WebSocketClientProtocol
+
+
+async def ws_receive_all_messages(ws: WebSocketClientProtocol, timeout: int = 15) -> list:
```

Comment:
move websocket specific functions to helpers/websockets. Mix async and sync functions in one modules is not a good practise

Reply:
Moved